bin/pylint_checkstyle.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import json
+import sys
+import subprocess
+import xml.dom.minidom
+
+
+def set_xml_attribute_from_dict(dom, xml_element, attribute_map):
+    for name, value in attribute_map.items():
+        attribute = dom.createAttribute(name)
+        attribute.value = value
+        xml_element.setAttributeNode(attribute)
+
+
+def main(file_names):
+    """
+    main entrypoint.
+    """
+    # checkstyle xml format is like
+    # <checkstyle>
+    #   <file name="foo/bar/baz.py">
+    #         <error line='7' column='120' severity='info'
+    #                message='Line is too long. [156/120]'
+    #                source='com.puppycrawl.tools.checkstyle.Metrics/LineLength'/>
+    #   </file>
+    # </checkstyle>
+    dom = xml.dom.minidom.Document()
+    root = dom.createElement('checkstyle')
+    dom.appendChild(root)
+    for f in file_names:
```

Comment:
INFO: Variable name &quot;f&quot; doesn&#x27;t conform to snake_case naming style